spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestViews.java

```diff
@@ -900,7 +906,7 @@ public void dropV1View() {
     String v1View = "v1ViewToBeDropped";
     sql("USE spark_catalog");
     sql("CREATE NAMESPACE IF NOT EXISTS %s", NAMESPACE);
-    sql("CREATE TABLE %s (id INT, data STRING)", tableName);
+    sql("CREATE TABLE IF NOT EXISTS %s (id INT, data STRING)", tableName);
```

Comment:
Hive is throwing table already found error. Even though we are switching catalog. we can also override this test for Hive.